src/Sulu/Component/Webspace/Manager/WebspaceCollectionBuilder.php

```diff
@@ -100,8 +101,8 @@ public function build()
         $this->portals = array();
         $this->portalInformations = array();
 
+        /** @var SplFileInfo $file */
```

Comment:
I guess there are different opinions on that. I would prefer to keep this in the foreach, because it's always like that in the Sulu codebase, and if it is the first line in the foreach it is more clear that this is the loop variable.